### PR TITLE
Add Nextflow schema

### PR DESCRIPTION
**Pull request by adamjtaylor**
Thank you @rxu17 for starting this! I have refined it slightly and think it is now in a place for us to merge.

### CODE REVIEW

**Comment by adamjtaylor** [comment]
Here is what the schema looks like in tower: 
Screenshot 2023-08-18 at 10 54 56

File: nextflow_schema.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        "miniatue_options": {
+            "title": "Miniature options",
+            "type": "object",
+            "fa_icon": "fas fa-palette",
+            "description": "Define HTAN Artist pipeline parameters.",
+            "properties": {
+                "remove_bg": {
+                    "type": "boolean",
+                    "description": "Remove non-tissue background",
+                    "default": true
```
Comment (rxu17) [inline nextflow_schema.json:37]: did we ever want to have the option to not remove background? (false)

**Review by rxu17 — APPROVED** [review]
LGTM! Just one question

File: nextflow_schema.json (hunk 1 above)
Comment (adamjtaylor) [reply nextflow_schema.json:37]: Yes, when set to false the background is kept.

**Pull request merged**